src/relayclient/HttpWrapper.ts

```diff
@@ -1,13 +1,15 @@
 import axios, { AxiosInstance, AxiosRequestConfig } from 'axios'
 
 const LOGMAXLEN = 120
+const DEFAULT_TIMEOUT = 15000
 
 export default class HttpWrapper {
   private readonly provider: AxiosInstance
   private readonly logreq: boolean
 
   constructor (opts: AxiosRequestConfig = {}, logreq: boolean = false) {
     this.provider = axios.create(Object.assign({
+      timeout: DEFAULT_TIMEOUT,
```

Comment:
Is there any chance at some point we will want to make this parameter configurable?

Reply:
yes, of course its configurable. in our test we override it with timeout:1